lib/src/editor/ui/editor.dart

```diff
@@ -93,13 +97,13 @@ class Editor extends StatelessWidget {
 }
 
 class _Canvas extends StatelessWidget {
-  Widget? component;
-  TransformationController _transformation = TransformationController();
+  final Widget? component;
   _Canvas(this.component);
   @override
   Widget build(BuildContext context) {
     return Consumer<ZoomProvider>(
       builder: (context, model, child) {
+        TransformationController _transformation = TransformationController();
```

Comment:
what required this to be relocated?

Reply:
I am passing in the controller into the two children